bitnami/wordpress/values.yaml

```diff
@@ -276,6 +276,10 @@ updateStrategy:
 ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
 ##
 schedulerName: ""
+## @param terminationGracePeriodSeconds In seconds, time the given to the %%MAIN_CONTAINER_NAME%% pod needs to terminate gracefully
```

Comment:
Could you please replace the placeholder `%%MAIN_CONTAINER_NAME%%`?

Reply:
It seems like there is a GitHub workflow called reame-generator-for-helm which is automatically replacing the string "WordPress" with %%MAIN_CONTAINER_NAME%% and commiting on my behalf. I tried disabling the workflow but it still runs upon my commit. Not sure how to deal with it, do you have an idea? I don't want to mess with the file because it shouldn't be inlcuded in the PR.